drone-fly-app/src/main/java/com/expediagroup/dataplatform/dronefly/app/service/listener/DefaultDroneFlyListener.java

```diff
@@ -0,0 +1,133 @@
+package com.expediagroup.dataplatform.dronefly.app.service.listener;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hive.metastore.MetaStoreEventListener;
+import org.apache.hadoop.hive.metastore.api.MetaException;
+import org.apache.hadoop.hive.metastore.events.AddPartitionEvent;
+import org.apache.hadoop.hive.metastore.events.AlterPartitionEvent;
+import org.apache.hadoop.hive.metastore.events.AlterTableEvent;
+import org.apache.hadoop.hive.metastore.events.CreateDatabaseEvent;
+import org.apache.hadoop.hive.metastore.events.CreateFunctionEvent;
+import org.apache.hadoop.hive.metastore.events.CreateTableEvent;
+import org.apache.hadoop.hive.metastore.events.DropDatabaseEvent;
+import org.apache.hadoop.hive.metastore.events.DropFunctionEvent;
+import org.apache.hadoop.hive.metastore.events.DropPartitionEvent;
+import org.apache.hadoop.hive.metastore.events.DropTableEvent;
+import org.apache.hadoop.hive.metastore.messaging.EventMessage.EventType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DefaultDroneFlyListener extends MetaStoreEventListener {
```

Comment:
```suggestion
public class LoggingDroneFlyListener extends MetaStoreEventListener {
```
(I think that's clearer that it just logs things and shouldn't really be used as a "default"?)

Reply:
`LoggingMetastoreListener` might be better coz it is actually a MetastoreListener.

Reply:
Agreed.